sdk/communication/azure-communication-networktraversal/swagger/src/main/java/CommunicationRelayCustomization.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import com.azure.autorest.customization.ClassCustomization;
+import com.azure.autorest.customization.Customization;
+import com.azure.autorest.customization.LibraryCustomization;
+import com.azure.autorest.customization.PackageCustomization;
+import org.slf4j.Logger;
+
+public class CommunicationRelayCustomization extends Customization {
+
+    @Override
+    public void customize(LibraryCustomization libraryCustomization, Logger logger) {
+        PackageCustomization models = libraryCustomization.getPackage("com.azure.communication.networktraversal.models");
+        models.getClass("CommunicationIceServer").getMethod("setUrls").setModifier(2);
+        models.getClass("CommunicationIceServer").getMethod("setUsername").setModifier(2);
+        models.getClass("CommunicationIceServer").getMethod("setRouteType").setModifier(2);
+        models.getClass("CommunicationIceServer").getMethod("setCredential").setModifier(2);
```

Comment:
Logic looks good to me, though I'd recommend using `Modifier.PRIVATE` instead of `2` as it will more clearly show the transformation being applied.

```suggestion
        models.getClass("CommunicationIceServer").getMethod("setUrls").setModifier(Modifier.PRIVATE);
        models.getClass("CommunicationIceServer").getMethod("setUsername").setModifier(Modifier.PRIVATE);
        models.getClass("CommunicationIceServer").getMethod("setRouteType").setModifier(Modifier.PRIVATE);
        models.getClass("CommunicationIceServer").getMethod("setCredential").setModifier(Modifier.PRIVATE);
```

Reply:
I removed the customization completely since the it's throwing an error saying that the setters are not used. So just putting them back to public. Can that still be good? @srnagar